### ISSUE TITLE
Appendix B - Project Manager and Product Owner

### ISSUE THREAD

**Issue by fmorelli** [WhiteHouse/playbook#80]
"Project Manager: Government manager of the Agile process. Also sometimes called Product Owner."

It would be wise to break out these two definitions. The roles are suitably different (and important). One can certainly note, in the case of smaller efforts, the role could be performed by one person.

To not have a Product Owner identified as a key government person is a significant impediment! Amongst other things, the Product Owner can not be administrative - it most have business expertise. They should, for example, be able to answer 80% of all questions asked by the development team within a few hours (either by knowing the answer or who to ask). Especially as the government looks to do Agile, it must understand this role within its organization and how it is key to successful solution delivery to the business.

**Comment by andersenpm** [comment]
I agree this revision should be made. The two might be the same person in certain circumstances, but the two roles are not synonymous. I would assume the Project Manager exists only through the duration of a project, but the Product Owner exists through the life of the product. And under the overall Product Owner, there might even be multiple Functional Owners. If the Project Manager is not someone who works closely with the functional side and the business rules, then the Product Owner should probably be someone else.
